packages/account/src/Components/currency-selector/currency-selector.tsx

```diff
@@ -266,7 +267,7 @@ const CurrencySelector = observer(
                                         )}
                                     </ThemedScrollbars>
                                 </Div100vhContainer>
-                                <Modal.Footer has_separator is_bypassed={is_mobile}>
+                                <Modal.Footer has_separator is_bypassed={isMobile}>
```

Comment:
Hey @fasihali-deriv , may i ask how are we handling tablet view scenarios in this and following components?